src/js/unfetch/index.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * This is required for the segment analytics
+ * It uses the unfetch library which is not compatbile with our fecth library and fails to initialize the
```

Comment:
A typo, missing what is failing to initialize.

Reply:
it's on the line above. Th segment analytics